app/src/app/model.ts

```diff
@@ -52,3 +52,23 @@ export interface Unit {
     visibility: string,
     description: string
 }
+
+export interface VotingRight {
+    id: string,
+    name: string,
+    votes: number
+  }
+
+export interface VotingRightWithPath extends VotingRight {
+    dbPath: string
+}
+
+export interface AvailableQuestion {
+    id: string,
+    name: string,
+    decretType: string,
+    byDelegateId: string,
+    byHouse: string,
```

Comment:
I would remove `house` naming, we don't use it anywhere. We use `votingRight` instead, right?

Reply:
Actually we are using it when we are filtering for the selection: https://github.com/davidvavra/duna-valka-assassinu/pull/53/files#diff-04808dcd2b691997f6f2d9dc57368787084dee1af305f3f27c467daa32dacfd6R78 And it is also needed for the "voice of landsraad" because at least last time the voice of Landsraad mentioned the name of the house which suggested the decret.

Reply:
Or do you just mean to rename "byHouse" to "byVotingRight"?

Reply:
Yes I mean `byVotingRight`

Reply:
Should be fixed.